views/random-beer.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each random}}
```

Comment:
In the `app.js` file on the `/random-beer` route when you are rendering the `random-beer.hbs` you are also passing the response from the API as `responseFromAPI`:

```
res.render('random-beer', { responseFromAPI });
```

So if you want to display the information that you get from the API you need to reference it with the same name as you're passing in the route:

```
{{#each responseFromAPI}}

  {{> beerpartial}}
  <h3>{{this.food_pairing}}</h3>
  <h3>{{this.brewer_tips}}</h3>

{{/each}}
```